### PR TITLE
[Delivers #95897486] add help/FAQ page

### PR DESCRIPTION
**Pull request by afeld**
With the changes from #399:

screen shot 2015-06-17 at 2 20 23 am

### CODE REVIEW

File: spec/requests/help_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+describe 'GET /help' do
+  it "displays successfully" do
+    get '/help'
+    expect(response.status).to eq(200)
+    expect(response.body).to include('FAQ')
```
Comment (cmc333333) [inline spec/requests/help_spec.rb]: Maybe verify that an anchor was created -- that the markdown was processed?

File: app/views/home/help.html.haml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.inset
+  - # TODO don't re-read file each time
+  = markdown(File.read('app/views/home/help.md'))
```
Comment (cmc333333) [inline app/views/home/help.html.haml]: Just checking -- does this need `RAILS_ROOT` to be portable?

**Comment by cmc333333** [comment]
Content makes sense to me

**Comment by afeld** [comment]
Updated! How would you feel about merging sooner than later, and iterating?

**Comment by phirefly** [comment]
:shipit:

**Pull request merged**